dashboard/src/components/Table/TestsTable.tsx

```diff
@@ -88,13 +138,95 @@ const TestsTable = ({
     </div>
   );
 
+  const onClickFilter = useCallback(
```

Comment:
nit: this use callback is just wasting space since you are just declaring an arrow function inside the jsx